ExtraRoles/ExtraRoles.cs

```diff
@@ -183,6 +185,8 @@ public void SetConfigSettings()
                 this.OfficerCD = HarmonyMain.OfficerKillCooldown.GetValue();
                 this.showEngineer = HarmonyMain.showEngineer.GetValue();
                 this.showJoker = HarmonyMain.showJoker.GetValue();
+                this.jokerCanDieToOfficer = HarmonyMain.jokerCanDieToOfficer.GetValue();
```

Comment:
This option does not exist anymore.

Reply:
Cheers, I knew the option changed, just tiredly botched the merge!